easybuild/easyconfigs/m/MotionCor2/MotionCor2-1.5.0-GCCcore-11.3.0.eb

```
@@ -0,0 +1,44 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Ake Sandgren
+# HPC2N
+# Umea University
+
+name = 'MotionCor2'
+version = '1.5.0'
+
+homepage = 'https://msg.ucsf.edu/'
+description = """MotionCor2 correct anisotropic image motion at the
+single pixel level across the whole frame, suitable for both single
+particle and tomographic images. Iterative, patch-based motion detection
+is combined with spatial and temporal constraints and dose weighting.
+
+Cite publication: Shawn Q. Zheng, Eugene Palovcak, Jean-Paul Armache,
+Yifan Cheng and David A. Agard (2016) Anisotropic Correction of
+Beam-induced Motion for Improved Single-particle Electron
+Cryo-microscopy, Nature Methods, submitted.
+BioArxiv: https://biorxiv.org/content/early/2016/07/04/061960
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+# No longer directly downloadable, available from https://msg.ucsf.edu/software
+sources = [
+    '%(name)s_%(version)s.zip',
+]
+checksums = ['cdb7b9aceb42d52e2a4c7a5e80270371376aea412d21fa1f1f28d4cc71c81834']
+
```

```suggestion
download_instructions = """Visit https://msg.ucsf.edu/software and fill in the form to download %s""" % sources[0]

```

Adding this means we get the following in the output in the fetching files step if you do not have that file downloaded:
```console
$ eb MotionCor2-1.5.0-GCCcore-11.3.0.eb -r
== fetching files...

Download instructions:

Visit https://msg.ucsf.edu/software and fill in the form to download MotionCor2_1.5.0.zip


```